packages/schema/lib/schemas/DynamicAuthInputFieldsSchema.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const AuthInputFieldSchema = require('./AuthInputFieldSchema');
+
+module.exports = makeSchema(
+  {
+    id: '/DynamicAuthInputFieldsSchema',
```

Comment:
Tests are providing an array of field objects for `AuthenticationSchema` (e.g. [{ key: 'access_token' }, { key: 'refresh_token' }]) so we need to do this to avoid “is not of a type(s) object” errors.

Reply:
```suggestion
    id: '/AuthFieldsSchema',
```

I suggest we don't use `Input` in the naming because there is no indication of input/output where it is used in the /AuthenticationSchema. Also, unlike for inputFields and outputFields, the `Dynamic` part is not necessary here because a function can't be used to define the auth fields.

Reply:
In the same vein, let's rename the `AuthInputFieldSchema` to `AuthFieldSchema`.

Reply:
Makes sense, I'll update!